app/bundles/CampaignBundle/Tests/Controller/CampaignControllerTest.php

```diff
@@ -77,4 +78,48 @@ public function testCampaignWithProject(): void
         $savedCampaign = $this->em->find(Campaign::class, $campaign->getId());
         Assert::assertSame($project->getId(), $savedCampaign->getProjects()->first()->getId());
     }
+
+    /**
+     * Test that campaign events include isRedirectTarget property when editing campaign.
+     */
+    public function testCampaignEditIncludesIsRedirectTargetProperty(): void
+    {
+        $campaign = new Campaign();
+        $campaign->setName('Test Campaign for Redirect Target');
+        $this->em->persist($campaign);
+
+        // Create a target event (the one being redirected to)
+        $targetEvent = new Event();
+        $targetEvent->setName('Target Event');
+        $targetEvent->setType('email.send');
+        $targetEvent->setEventType('action');
+        $targetEvent->setCampaign($campaign);
+        $this->em->persist($targetEvent);
+
+        // Create a source event that redirects to the target
+        $sourceEvent = new Event();
+        $sourceEvent->setName('Source Event');
+        $sourceEvent->setType('campaign.jump_to_event');
+        $sourceEvent->setEventType('action');
+        $sourceEvent->setCampaign($campaign);
+        $sourceEvent->setDeleted();
+        $sourceEvent->setRedirectEvent($targetEvent);
+        $this->em->persist($sourceEvent);
+
+        $this->em->flush();
+        $this->em->clear();
+
+        // Request the edit page for the campaign
+        $this->client->request('GET', '/s/campaigns/edit/'.$campaign->getId());
+        $clientResponse = $this->client->getResponse();
+
+        Assert::assertSame(Response::HTTP_OK, $clientResponse->getStatusCode());
```

Comment:
```suggestion
        $this->assertResponseIsSuccessful();
```
Simpler and gives more debug details if fails

Reply:
Done.